### PR TITLE
blog: Self-hosted starter limit changes

### PR DESCRIPTION
**Pull request by ZJvandeWeg**
## Description

Part of: https://github.com/FlowFuse/admin/issues/275
## Checklist

 - [ ] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

**Review by MarianRaphael — APPROVED** [review]

**Comment by MarianRaphael** [comment]
See also: https://github.com/FlowFuse/website/pull/1783

File: src/blog/2024/03/flowfuse-self-hosted-starter-resource-limits.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+title: FlowFuse Open Source Starter Tier Resource Limits
+subtitle: FlowFuse was built to empower Node-RED to run everywhere, even behind a firewall
+description: FlowFuse was built to empower Node-RED to run everywhere, even behind a firewall
```
Comment (knolleary) [inline src/blog/2024/03/flowfuse-self-hosted-starter-resource-limits.md:4]: Needs updating from the previous blog post

Comment (ZJvandeWeg) [inline src/blog/2024/03/flowfuse-self-hosted-starter-resource-limits.md:4]:
```suggestion
subtitle: The first 5 Node-RED runtimes are part of the starter package going forward
description: The first 5 Node-RED runtimes are part of the starter package going forward
```

**Comment by ZJvandeWeg** [comment]
@knolleary I was unaware that was due this week, and thus it's not part of this PR. @MarianRaphael This shouldnb've been communicated with all stakeholders and ensure everyone is on the same page.

**Comment by knolleary** [comment]
It had been discussed in the Architecture/Product meeting last week. The change request for it was raised earlier this week.

As this is entirely configuration changes in the platform, it does not need to be tied to publishing 2.2 (unlike the licensing changes which very much is).

So if there's any doubt, we can hold off that change today - focus on the licensing changes and then make sure we have full alignment with all stakeholders with the pricing changes next week.

**Comment by knolleary** [comment]
Raised artwork request issue: https://github.com/FlowFuse/website/issues/1788

**Comment by ZJvandeWeg** [comment]
We can update the art work later, but I feel it's weird not to talk about changes which have been published.

**Pull request merged**